src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -541,37 +546,47 @@ public void AssemblyLoad()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/18718", TargetFrameworkMonikers.Uap)] // Need to copy files out of execution directory'
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Does not support Assembly.LoadFile")]
         public void AssemblyResolve()
         {
             CopyTestAssemblies();
 
             RemoteExecutor.Invoke(() => {
-                ResolveEventHandler handler = (sender, e) =>
+                bool AssemblyResolveFlag = false;
+                ResolveEventHandler handler = (sender, args) =>
                 {
+                    Assert.Same(AppDomain.CurrentDomain, sender);
+                    Assert.NotNull(args);
+                    Assert.NotNull(args.Name);
+                    Assert.NotNull(args.RequestingAssembly);
+                    AssemblyResolveFlag = true;
                     return Assembly.LoadFile(Path.Combine(Environment.CurrentDirectory, "AssemblyResolveTestApp.dll"));
                 };
 
                 AppDomain.CurrentDomain.AssemblyResolve += handler;
 
                 Type t = Type.GetType("AssemblyResolveTestApp.Class1, AssemblyResolveTestApp", true);
                 Assert.NotNull(t);
+                Assert.True(AssemblyResolveFlag);
```

Comment:
I was surprised this didn't fail in the first run.  Added this extra check to be sure the handler is run....

Reply:
This test is not running the handler...

Reply:
Added a test which will run the handler and created issue #38361